css/dashboard.manage.css

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file
+ * Styling for contacts module dashboard management.
+ */
+
+/* Tab appearance. */
```

Comment:
Should `contacts.icons.css` and `dashboard.manage.css` be combined into `contacts.toolbar.css`?